frontend/mobilityai_react/App/Screens/PatientData.js

```diff
@@ -0,0 +1,286 @@
+            .innerRadius(60); //the size of the inner 'donut' whitespace
+
+        return (
+            <ScrollView>
+                <View>
+                    <Text style={styles.titleFont}>{firstName + " " + lastName}</Text>
+
+                    <View style={styles.textInline}>
+                        <Text style={styles.center}>Daily User Activity</Text> 
+                        <GetDate />
+                    </View>
+
+                    {/* Displaying the pie chart of all the activities */}
+                    <View style={styles.center}>
+                        <Surface width={width} height={height}>
+                            <Group x={width / 2} y={height / 2}>
+                                {
+                                    sectionAngles.map(section => (
+                                        <Shape
+                                            key={section.index}
+                                            d={path(section)}
+                                            stroke="#000"
+                                            fill={colours[section.index]}
+                                            strokeWidth={1}
+                                        />
+                                    ))
+                                }
+                            </Group>
+                        </Surface>
+                    </View>
+
+                    {/* Displaying the circle buttons for each activity */}
+                    <View style={styles.flexDir}>
+                        <TouchableHighlight onPress={this._onPressButton.bind(this, 0, data)} underlayColor="white">
+                            <Circle activity='Standing' activityIcon='male' iconLib='font-awesome' color={colours[0]} />
+                        </TouchableHighlight>
+
+                        <TouchableHighlight onPress={this._onPressButton.bind(this, 1, data1)} underlayColor="white">
+                            <Circle activity='Sitting' activityIcon='airline-seat-recline-normal' iconLib='MaterialIcons' color={colours[1]} />
+                        </TouchableHighlight>
+
+                        <TouchableHighlight onPress={this._onPressButton.bind(this, 2, data2)} underlayColor="white">
+                            <Circle activity='Lying Down' activityIcon='bed' iconLib='font-awesome' color={colours[2]} />
+                        </TouchableHighlight>
+
+                        <TouchableHighlight onPress={this._onPressButton.bind(this, 3, data)} underlayColor="white">
+                            <Circle activity='Walking' activityIcon='directions-walk' iconLib='MaterialIcons' color={colours[3]} />
+                        </TouchableHighlight>
+
+                        <TouchableHighlight onPress={this._onPressButton.bind(this, 4, data2)} underlayColor="white">
+                            <Circle activity='Miscellaneous' activityIcon='question' iconLib='font-awesome' color={colours[4]} />
```

Comment:
Since these colours are associated to activities, it would be a lot easier to understand if they were stored in a dictionary eg: `color={colours['walking']}`